src/Components/Layout/Navbar.tsx

```diff
@@ -1,44 +1,76 @@
 import React from "react"
-import { 
+import {
   HStack,
+  IconButton,
   Image,
   Flex,
   Link,
   LinkBox,
   LinkOverlay,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
 } from '@chakra-ui/react'
 import { Link as GatsbyLink } from 'gatsby'
+import { HamburgerIcon } from '@chakra-ui/icons'
+import Logo from '../../../static/Images/logo.png'
 
 export const Navbar = (): JSX.Element => {
   return (
-    <Flex alignItems='center' bgColor='purple' px={16} py={12} justifyContent='space-between'>
+    <Flex alignItems='center' bgColor='purple' px={['24px', '64px']} py={['24px', '48px']} justifyContent='space-between'>
       <LinkBox as={HStack}>
-        <Image src='../../../Images/Logos/ColorLogo.jpg' boxSize='45px'/>
+        <Image src={Logo} boxSize='44px'/>
         <LinkOverlay 
           as={GatsbyLink}
           to='/'
           color='white'
-          fontSize={32}
+          fontFamily='Helvetica'
+          fontSize={[24, 32]}
           fontWeight='bold'
           _hover={{
             textDecoration: 'underline',
             textDecorationColor: 'orange',
           }}
         >
-          Internet for All NYC
+          Internet for All
         </LinkOverlay>
       </LinkBox>
-      <HStack spacing={6}>
-        <Link as={GatsbyLink} to='/principles' variant='nav'>
+      <HStack spacing='24px' display={['none', 'block']}>
+        <Link as={GatsbyLink} to='/principles' variant='navbar'>
           Principles
         </Link>
-        <Link as={GatsbyLink} to='/coalition-partners' variant='nav'>
+        <Link as={GatsbyLink} to='/coalition-partners' variant='navbar'>
           Coalition Partners
         </Link>
-        <Link as={GatsbyLink} to='/take-action' variant='nav'>
+        <Link as={GatsbyLink} to='/take-action' variant='navbar'>
           Take Action
         </Link>
       </HStack>
+      <Menu>
+        <MenuButton backgroundColor='purple'
+          as={IconButton}
+          icon={<HamburgerIcon color='white'/>}
+          display={['block', 'none']}
```

Comment:
Chakra's `Menu` component doesn't have a `display` prop so I had the it's children display only at the smallest breakpoint for mobile.